Add tuna to list of bootstrap nodes

robertfoss opened the pull request.

robertfoss committed:
Add tuna to list of bootstrap nodes

str4d commented:
Awesome, thanks!

Build is failing because of recent Travis CI changes. I'll push a commit that fixes it.

str4d committed:
Travis CI: Use Trusty build environment

Travis CI recently bumped the default build environment from Trusty to
Xenial. The latter does not enable installing Java 7 or 8, which we
still test against.
